app/pillar/handler/crud.go

```diff
@@ -101,3 +101,53 @@ func DeleteSearch(c *web.AppContext) {
 	err := service.DeleteSearch(c)
 	doRespond(c, nil, err)
 }
+
+func CreateUpdateForm(c *web.AppContext) {
```

Comment:
Do you think it makes sense to add comments before each func to have a better idea of they do?